examples/install_with_clone.sh

```diff
@@ -71,34 +90,26 @@ mysql < sql/0.5.0.sql
 mysql < sql/0.6.0.sql
 mysql < sql/0.7.0.sql
 
+PASSWD=""
 
 cp examples/lxd_manager.conf /etc/apache2/sites-available/
 
-pm2 start node/events.js
+pm2 start node/events.js || exit $?
 
-pm2 startup
+pm2 startup || exit $?
 
-pm2 save
+pm2 save || exit $?
 
 # Add cron job for gathering data
-crontab -l | { cat; echo "*/5 * * * * php /var/www/LxdMosaic/src/cronJobs/fleetAnalytics.php"; } | crontab -
+crontab -l 2>/dev/null | { cat; echo "*/5 * * * * php /var/www/LxdMosaic/src/cronJobs/fleetAnalytics.php"; } | crontab -
```

Comment:
`2>/dev/null` why was this added to this line, but not the line below ?

Have you made sure the containers run these cron jobs ?

Reply:
crontab -l puts a warning on stderr when the user (root) has no crontab (which is standard for a fresh install). 2>/dev/null just hides that and it's not needed on the 2nd line because by then there is a crontab (created by the first line)

Yup, the cron tasks are install and do run. I connected each of the containers to the host LXD instance to do some quick testing as well.

I should mention that v0.8 has a couple of moderate level npm audit issues

Reply:
Ah okay that makes sense! 

There is only 1 low alert on the latest branch!